packages/fyllut/src/components/AllForms.test.jsx

```diff
@@ -0,0 +1,54 @@
+import { render, screen } from "@testing-library/react";
+import fetchMock from "jest-fetch-mock";
+import React from "react";
+import { MemoryRouter } from "react-router-dom";
+import { AllForms } from "./AllForms";
+
+describe("AllForms", () => {
+  it("Show loading when fetching forms from backend", async () => {
+    fetchMock.mockImplementation((url) => {
+      return Promise.resolve(new Response(JSON.stringify([])));
+    });
+
+    render(
+      <MemoryRouter initialEntries={["/fyllut"]}>
+        <AllForms />
+      </MemoryRouter>
+    );
+
+    expect(screen.getByRole("heading", { name: "Laster..." })).toBeInTheDocument();
+  });
+
+  it("Show no form founded when there is no form from backend", async () => {
+    fetchMock.mockImplementation((url) => {
+      return Promise.resolve(new Response(JSON.stringify([])));
+    });
+
+    render(
+      <MemoryRouter initialEntries={["/fyllut"]}>
+        <AllForms />
+      </MemoryRouter>
+    );
+    expect(await screen.findByRole("heading", { name: "Finner ingen skjemaer" })).toBeInTheDocument();
```

Comment:
Test if combining this test and the "Laster..."-test above will remove ACT-warnings